pages/common/d2.md

```diff
@@ -0,0 +1,28 @@
+# d2
+
+> A modern diagram scripting language that turns text to diagrams.
+> More information: <https://d2lang.com/tour/intro/>.
```

Comment:
```suggestion
> More information: <https://d2lang.com/tour/man>.
```
Maybe, we can update this link to the man page of the command instead.

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links